pkg/logging/logging.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
The year is now 2018.

Reply:
Not in Canada, typical American assumptions.

(will fix)